boring/src/ssl/test/ech.rs

```diff
@@ -0,0 +1,62 @@
+use crate::hpke::HpkeKey;
+use crate::ssl::ech::SslEchKeys;
+use crate::ssl::test::Server;
+use crate::ssl::HandshakeError;
+
+static ECH_CONFIG_LIST: &[u8] = include_bytes!("../../../test/echconfiglist");
```

Comment:
Can you add a comment explaining how to regenerate those files?

Reply:
Done